### PR TITLE
Updates viscous dissipation docstring

### PR DESCRIPTION
**Pull request by glwagner**
Not a very important PR, but some PRs are like that.

### CODE REVIEW

File: src/TurbulenceStatistics/viscous_dissipation.jl [hunk 1]
```diff
@@ -21,6 +21,21 @@ kinetic energy of `model`.
 
 Calling `compute!(ϵ::ViscousDissipation)` computes the viscous dissipation of
 turbulent kinetic energy associated with `model` and stores it in `ϵ.data`.
+
+The viscous dissipation ``\epsilon`` is defined
+
+```math
+ϵ ≡ ∫ uⱼ ∂ⱼ τᵢⱼ \, \rm{d} V
+```
```
Comment (ali-ramadhan) [inline src/TurbulenceStatistics/viscous_dissipation.jl:29]: Tests are failing and complaining about invalid escape sequences. I think we need to escape `\` in docstrings: https://juliadocs.github.io/Documenter.jl/stable/man/latex/#Escaping-Characters-in-Docstrings

````suggestion
The viscous dissipation ``\\epsilon`` is defined

```math
ϵ ≡ ∫ uⱼ ∂ⱼ τᵢⱼ \\, \\rm{d} V
```
````

**Review by ali-ramadhan — APPROVED** [review]

**Comment by ali-ramadhan** [comment]
@glwagner Should we merge this PR?

**Review by navidcy — APPROVED** [review]

**Comment by glwagner** [comment]
hmmm yes but conflicts, dang

**Comment by glwagner** [comment]
lets do #124 first

**Comment by glwagner** [comment]
Oh, we don't have this function anymore cause its in `Oceanostics`.

**Pull request closed without merging**